projects/angular/src/modal/modal-body.ts

```diff
@@ -35,6 +38,17 @@ export class ClrModalBody implements OnDestroy {
     });
   }
 
+  ngAfterViewChecked(): void {
```

Comment:
Why do you think we should run this after every view checked? This would likely be something you set once unless the browser is resized.

Reply:
Sometimes a dynamic grid or a dynamic form is put inside a dialog, then the size is not fixed on user's actions.